app/models/container_deployment.rb

```diff
@@ -6,5 +6,314 @@ class ContainerDeployment < ApplicationRecord
   has_many :container_volumes, :as => :parent
   has_many :custom_attributes, :as => :resource, :dependent => :destroy
   has_many :authentications, :as => :resource, :dependent => :destroy
-  serialize :customize, Hash
+  serialize :customizations, Hash
+  AUTH_CONSTS = {"AuthenticationAllowAll" => AuthenticationAllowAll, "AuthenticationHtpasswd" => AuthenticationHtpasswd, "AuthenticationLdap" => AuthenticationLdap, "AuthenticationRequestHeader" => AuthenticationRequestHeader, "AuthenticationOpenId" => AuthenticationOpenId, "AuthenticationGoogle" => AuthenticationGoogle, "AuthenticationGithub" => AuthenticationGithub, "AuthPrivateKey" => AuthPrivateKey, "AuthenticationRhsm" => AuthenticationRhsm}.freeze
+  AUTHENTICATIONS_NAMES = {"AuthenticationAllowAll" => "AllowAllPasswordIdentityProvider", "AuthenticationHtpasswd" => "HTPasswdPasswordIdentityProvider", "AuthenticationLdap" => "LDAPPasswordIdentityProvider", "AuthenticationRequestHeader" => "RequestHeaderIdentityProvider", "AuthenticationOpenId" => "OpenIDIdentityProvider", "AuthenticationGoogle" => "GoogleIdentityProvider", "AuthenticationGithub" => "GitHubIdentityProvider", "AuthPrivateKey" => "ssh", "AuthenticationRhsm" => "rhsm"}.freeze
```

Comment:
Minor style, but this would be cleaner with one-per-line.  Also, why is the first line needed at all... str.safe_constantize should give you the same thing (unless you're whitelisting?)